pulp_ansible/app/viewsets.py

```diff
@@ -167,10 +167,15 @@ def sync(self, request, pk):
         serializer = RepositorySyncURLSerializer(data=request.data, context={"request": request})
         serializer.is_valid(raise_exception=True)
         repository = serializer.validated_data.get("repository")
+        mirror = serializer.validated_data.get("mirror", False)
```

Comment:
With this specified here can we de-duplicate the default by making the signature of the sync task:

`def sync(remote_pk, repository_pk, mirror):`